code/game/objects/structures/banking/banking.dm

```diff
@@ -0,0 +1,209 @@
+/**
+ * Everything to do with international banking
+ * 
+ * SPONSORED by the Foreign Nations Alliance (FNA)!
+ */
+
+/obj/structure/banking
+    name = "banking"
+    desc = "debug"
+    icon = 'icons/obj/banking.dmi'
+    icon_state = ""
+    layer = 3
+    anchored = TRUE
+    density = TRUE
+    flammable = FALSE
+    not_movable = FALSE
+    not_disassemblable = TRUE
+
+/obj/structure/banking/computer
+    var/operatingsystem = "FNB OS 42-G7"
+    var/display = "FNB OS 1.0.0"
+    var/mainbody = ""
+	var/mainmenu = ""
+	var/mob/user
+
+/obj/structure/banking/computer/atm
+    name = "ATM"
+    desc = "an Automatic Teller Machine (ATM), owned by the Foreign Nations Bank, founded by the Foreign Nations Alliance. This ATM allows for international banking and overall storing your money in a safe place."
```

Comment:
overall storage of money.

client perspective its a description of the structure, shouldn't reference them, as we could be playing a non-human

Reply:
I'll let a cockroach bank at FNB anyway. It's money

Reply:
Humans only though